packages/didcomm/src/modules/message-pickup/MessagePickupModuleConfig.ts

```diff
@@ -36,8 +38,12 @@ export interface MessagePickupModuleConfigOptions<MessagePickupProtocols extends
 export class MessagePickupModuleConfig<MessagePickupProtocols extends MessagePickupProtocol[]> {
   private options: MessagePickupModuleConfigOptions<MessagePickupProtocols>
 
+  #messagePickupRepository: MessagePickupRepository
+
   public constructor(options: MessagePickupModuleConfigOptions<MessagePickupProtocols>) {
     this.options = options
+    // Message Pickup queue: use provided one or in-memory one if no injection symbol is yet defined
+    this.#messagePickupRepository = options.messagePickupRepository ?? new InMemoryMessagePickupRepository()
```

Comment:
Should we change this to something like messageQueueRepository? or queuedMessageRepository?

Reply:
This subtle question made me think that the approach we are following is not correct: this message queue is part of DIDComm core, regardless of the message pickup mechanism we use (for instance, the infamous "implicit" mode present in some mediators do not use Message Pickup protocol at all, but they do access this element to send queued messages to their clients at the moment they connect to a WebSocket).

In such case, this [QueueTransportMessageRepository](https://github.com/decentralized-identity/didcomm-messaging/blob/main/extensions/return_route/main.md#queue-transport) must be defined in the base DIDComm module configuration, and therefore much of the things done in this PR would not make sense.

Reply:
I agree. Routing/forwarding/queuing is part of core. Mediator management and pickup not! :)